agr_java_core/src/main/java/org/alliancegenome/api/entity/DiseaseAnnotationDocument.java

```diff
@@ -22,13 +22,13 @@
 public class DiseaseAnnotationDocument extends SearchableItemDocument {
 
 	private String uniqueId;
-	private VocabularyTerm diseaseRelation;
+	private VocabularyTerm relation;
 	private DOTerm object;
 	private List<ECOTerm> evidenceCodes;
 	private Reference singleReference;
 	private Set<Reference> references;
 
-	private String diseaseRelationNegation;
+	private String negationInclRelation;
```

Comment:
@markquintontulloch what does "Incl" mean? Include? if so just spell it out?

Reply:
@oblodgett negationInclusiveRelation, but I found that to be a bit too long as a variable name. Incl seems common enough as an abbreviation to me to be acceptable here.

Reply:
Ask @mluypaert, but I assume so!  Can't say I'm a huge fan of the new field name - I'd have preferred relationIncludingNegation, or just relationText, as it's a string describing the relation.

Reply:
Ya I am in favor of the relationText also.

Reply:
@markquintontulloch The problem with using "Text" as a description is that is doens't explain anything about the content, nor does this string represent a display text, this string only has an additional `_not_` inserted into the relation string if it is negated, while the display text would essentially be replacing all _ with spaces.

Reply:
is the field ALWAYS the negated version of the relation? Maybe thats the confusion here?

Reply:
@oblodgett no, only when negated is true, see the code that generates it:
https://github.com/alliance-genome/agr_java_software/pull/1130/files#diff-8272ea294f7bd959d0f02f20fe9d495da2a40ad9e9a8669bbfaa71cfbb5402f2L164-R167

Reply:
@mluypaert right thats what I suspected. So I would just call it that "relationText".

Reply:
Not sure what you mean by it not representing a display text.  Isn't that exactly what it is - the text that is displayed for a relation, which takes the negated field into account?

Reply:
ya... I would be fine with displayRelationText also... which tells me its generated.

Reply:
@markquintontulloch @oblodgett  No, because the text being displayed to users (in the website) is `is implicated in` or `is NOT implicated in`, while the variable here will hold `is_implicated_in` or `is_not_implicated_in`. Hence it's closer to the relation value than it is to the display text.

Reply:
This is not text, merely the relation value including the negation property.

Reply:
@oblodgett @markquintontulloch Does `relationNegationMerge` sound any better (than `negationInclRelation` or `negationInclusiveRelation`)?

Reply:
It doesn't need to include "negation". From my point if we generate something from obsolete or negated or whatever criteria that doesn't need to be included in the name.... just a way that the name indicates that its a generated field.

Reply:
I'd go with something like `relationString` or `generatedRelationString` if you're not happy with `relationText`.

Reply:
So I do agree that "relationText" would be misleading if we are going to modify it to include other things. So to me "generatedRelationText" would be best then the variable name is independent of content.